contracts/games/abstract/FutballCardGame.sol

```diff
@@ -0,0 +1,69 @@
+pragma solidity 0.5.0;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity/contracts/lifecycle/Pausable.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../../IFutballCardsAttributes.sol";
+
+contract FutballCardGame is Ownable, Pausable {
+    using SafeMath for uint256;
```

Comment:
This is an abstract contract that aims to implement common functionality between the 2 player games HeadToHead and MatchPrediction